meson.build

```diff
@@ -56,7 +56,8 @@ jo1 = custom_target('jo.1',
                     command: [pandoc_commands, '@OUTPUT@', '@INPUT@']).full_path()
 run_command(pandoc_commands,
             join_paths(meson.current_build_dir(), 'jo.1'),
-            join_paths(meson.current_source_dir(), 'jo.pandoc'))
+            join_paths(meson.current_source_dir(), 'jo.pandoc'),
+            check: false)
```

Comment:
The point of Meson adding the warning was specifically to make people consider whether they need to check the return value, so choosing "false" based purely on it being the old default seems wrong.

Let's instead look at the command in question and ponder: do we need to check this?

What happens if the pandoc command totally fails? Should this be an error? Probably.